CPM/Collaborative-Mark-Policy-draft.md

```diff
@@ -171,6 +179,9 @@ You need a trademark licence if you want to make merchandise with the OSM marks
 * use OSM's logos without modifying them; and
 * truthfully advertise to customers how much of the selling price, if any, will be donated to OSM projects.
 
+### 4.5. Fundraising 
+You need a trademark licence if you are are referring to OpenStreetMap in the context of fundraising in any form. 
```

Comment:
I would say maybe "if you are using or referring to the OpenStreetMap marks to fundraise in any way"? But that's mostly stylistic.